nodes/src/nodes/index_search/IGlobal.py

```
@@ -331,32 +327,22 @@ def _load_search_flags(self, connConfig: Dict[str, Any]) -> None:
         if match_operator_raw not in (*VALID_MATCH_OPERATORS, ''):
             warning(f"matchOperator must be 'and', 'or', or 'exact'; got: {match_operator_raw}")
             match_operator_raw = ''
         self.search_match_operator = match_operator_raw or 'or'
 
         try:
             slop_val = connConfig.get('slop')
-            self.search_exact_slop = (
-                int(slop_val or 0) if self.search_match_operator == 'exact' else 0
-            )
+            self.search_exact_slop = int(slop_val or 0) if self.search_match_operator == 'exact' else 0
         except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)